### PR TITLE
Sophie docs

### PR DESCRIPTION
**Pull request by sophiehe**
broken link edit in docs

### CODE REVIEW

**Comment by zenweasel** [comment]
@sophiehe Looks like there are some conflicts that need to be resolved

**Comment by zenweasel** [comment]
Looks like these screenshots are of the old dashboard?

File: admin/revisions.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Payment
```
Comment (zenweasel) [inline admin/revisions.md]: Why is this info in the revisions doc?

Comment (aaronjudd) [inline admin/revisions.md]: and what about revisions?

**Pull request merged**